### ISSUE TITLE
TEmbeddedController

### ISSUE THREAD

**Issue by bigbear2** [WobbyChip/Delphi#1]
Hi, I am very interested in the MSIControl project because on windows 11 MSI Dragon Center blocks my pc. The only program that works with my laptop is yours. I also program in Pascal (Delphi \ Lazarus) and I wanted to know if it was possible to have the TEmbeddedController pas to manage the fans. I hope to hear from you
PS: Sorry for my English

**Comment by WobbyChip** [comment]
Yes, https://cdn.discordapp.com/attachments/730504664625184830/941291606584209438/EmbeddedController.zip

**Comment by bigbear2** [comment]
Thank you so much. Anything in delphi you can ask for
